OutOfSchool/OutOfSchool.IdentityServer/OutOfSchool.IdentityServer.csproj

```diff
@@ -30,6 +30,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.1" />
     <PackageReference Include="Microsoft.Extensions.Identity.Core" Version="6.0.6" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.8.0" />
+    <PackageReference Include="NetEscapades.AspNetCore.SecurityHeaders" Version="0.18.0" />
```

Comment:
if we import it to common, we automatically have it in every project that depends on common. No need to have it here